iceoryx_examples/iceperf/roudi_main_static_config.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/iceoryx_posh_config.hpp"
+#include "iceoryx_posh/iceoryx_posh_types.hpp"
+#include "iceoryx_posh/internal/log/posh_logging.hpp"
+#include "iceoryx_posh/roudi/iceoryx_roudi_app.hpp"
+#include "iceoryx_posh/roudi/roudi_cmd_line_parser_config_file_option.hpp"
+
+int main(int argc, char* argv[])
+{
+    using iox::roudi::IceOryxRouDiApp;
+
+    iox::config::CmdLineParserConfigFileOption cmdLineParser;
+    cmdLineParser.parse(argc, argv);
+
+    iox::RouDiConfig_t roudiConfig;
+    // roudiConfig.setDefaults(); can be used if you want to use the default config only.
+
+    /// @brief Create Mempool Config
+    iox::mepoo::MePooConfig mepooConfig;
+
+    /// @details Format: addMemPool({Chunksize(bytes), Amount of Chunks})
+    mepooConfig.addMemPool({128, 10000}); // bytes
+    mepooConfig.addMemPool({1024, 5000});
```

Comment:
this is also already a kilobyte ;)
Could you maybe create a constexpr for kbytes and Mbytes?